package.json

```diff
@@ -14,7 +14,8 @@
   },
   "dependencies": {
     "bindings": "^1.5.0",
-    "node-addon-api": "^1.7.1"
+    "node-addon-api": "^1.7.1",
+    "node-gyp": "latest"
```

Comment:
I figured using `latest` is effectively equivalent to not having it in `dependencies` at all. Is this ok?